talent/hiring_process.md

```diff
@@ -0,0 +1,78 @@
+# Hiring Process Explained
+
+This document explains the internal hiring process at source{d} and the responsibilities for each person related to it.
+
+
+### Job Description
+
+- Job Description is written by the lead engineers with the Talent's Manager support. 
+
+- For every meeting between the lead and Anna there will be a PR in the [minutes](https://github.com/src-d/minutes) repo and the job descriptions will be uploaded at the private repository https://github.com/src-d/company.
+
+- CTO and CEO should be added as reviewers (but should not be bottlenecks to continue).
+
+- Anna posts the job description through Lever on the source{d} website.
+
+### Sourcing
+
+- Lead engineers are responsible to identity people to be contacted and other sourcing channels such as niche job boards. 
+
+- At any given moment the Talent Manager is able to drop in and work on finding more developers.
+
+- Developer lists/sourcing channels should continuously be defined and refreshed after communication between the Talent Manager and the leads. 
+
+- The Talent Manager is responsible to work with the Developer Community team in order to gain people through community actions (Twitter/Blog etc.). 
+
+### First Contact
+
+- Any developer should be contacted by the lead engineers or under their name at least (after together defining the outgoing message). In case the Talent Manager is contacting them, it should be done from the lead's email acccount. It's a motivating factor for the developers to be contacted by the lead engineers. 
```

Comment:
I'm not entirely comfortable having mails sent using my mail account. I find it weird...

Reply:
Not a strong opinion, but personally that is a bit strange for me as well. At this point, applicant would have known nothing about a lead or a team, so I'm not sure how motivating can be to know that somebody, you will be doing technical work with, is spending time sending emails.

At the same time I'm enthusiastic 👍 for leads proactively reaching out to potential candidates they know about.

Reply:
@rporres very simple, you send them out yourself. This is purely a support offer from @tsolakoua in case you don't have the time and want to reach out to a larger number of people with similar messaging.

Reply:
If you have a strong opinion about mails coming from the lead, I can live with that. But I agree with @bzz about the doubts of how motivating can be to receive mails from the lead. I do agree that it is that a phone screen with the lead is certainly motivating but I don't feel the same with mails.

Reply:
I'd rather have emails sent actually by the leads, or if they're not able to do so by someone else in their names (but not using their names)